style.css

```diff
@@ -0,0 +1,156 @@
+* {
+  margin: 0;
+}
+
+header {
+  border: 3px solid black;
+  padding: 10px 20px;
+  align-self: stretch;
+}
+
+h2 {
+  align-self: center;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+
+  /* gap: 28px; */
+  align-items: center;
+  font-weight: 600;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
+  font-size: 20px;
+  font-weight: 700;
+}
+
+nav ul li {
+  padding-left: 20px;
+  padding-right: 20px;
+  cursor: pointer;
+}
+
+nav ul li:nth-child(even) {
+  border-right: 2px black solid;
+  border-left: 2px black solid;
+}
+
+.book-item {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 2%;
+  padding-right: 2%;
+  margin: 0 auto;
+}
+
+.book-item:nth-child(odd) {
+  background-color: #dcdcdc;
+}
+
+.main {
+  display: flex;
+  max-width: 100%;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 15px;
+  padding: 10px;
+}
+
+.books-list {
+  align-self: stretch;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+}
+
+#books-container {
+  align-self: stretch;
+  margin-left: 5%;
+  margin-right: 5%;
+}
+
+hr {
+  background-color: black;
+  width: 15%;
+  height: 2px;
+}
+
+.add-book h2 {
+  align-self: center;
+}
+
+form {
+  margin-top: 40px;
+}
+
+form button {
+  float: right;
+  background-color: white;
+  border-right: 3px solid black;
+  border-bottom: 3px solid black;
+}
+
+.book-item button {
+  background-color: white;
+  border-right: 5px solid black;
+  border-bottom: 5px solid black;
+  margin-top: 2px;
+  margin-bottom: 2px;
+}
+
+form input {
+  border: 2px solid black;
+  width: 50vw;
+  padding: 3px 8px;
+}
+
+.section {
+  display: none;
+}
```

Comment:
@AnselemOdims 
Thank you sir for the detailed observation. It will certainly increase my knowledge. The line of the code you have maintained is about to change. When I change the display property It shows different behavior. It shows this.


![2022-03-24_11-22-47](https://user-images.githubusercontent.com/78845635/159855003-c0b2447f-b824-467e-914a-72cb5a1f8492.jpg)


Can you please help me, what should I change so that it gets fixed?
Looking forward to hearing from you!

Reply:
I noticed that not importing `luxon` properly is what is causing the errors. Please ensure it is properly imported.